scalameta/parsers/src/main/scala/scala/meta/internal/parsers/Messages.scala

```diff
@@ -22,4 +22,7 @@ object Messages {
     }
     QuasiquoteRankMismatch(rank, rank - 1, hint)
   }
+
+  def IllegalCombinationModifiers(mod1: Mod, mod2: Mod): String =
+    s"illegal combination of modifiers: $mod1 and $mod2"
```

Comment:
Scala parser says something like:

```
scala> final sealed class T
<console>:35: error: illegal combination of modifiers: final and sealed for: class T
       final sealed class T
                          ^
```

How difficult would it be to add the missing `for` clause?